htdocs/adherents/card.php

```diff
@@ -1969,6 +1980,13 @@ function initfieldrequired() {
 					}
 				}
 
+				// Delete Member
```

Comment:
In Dolibarr look and feel convention, button action that close or delete are always at end of list

Reply:
okay